pkgs/top-level/perl-packages.nix

```diff
@@ -5831,13 +5832,13 @@ let
 
   DBIxClass = buildPerlPackage {
     pname = "DBIx-Class";
-    version = "0.082842";
+    version = "0.082841";
```

Comment:
oops, it was pinned too
but 0.082842 breaks `Catalyst-Authentication-Store-DBIx-Class`